dd-java-agent/appsec/src/main/java/com/datadog/appsec/gateway/GatewayBridge.java

```diff
@@ -159,6 +161,31 @@ private Flow<Void> onNetworkConnection(RequestContext ctx_, String url) {
     }
   }
 
+  private Flow<Void> onFileLoaded(RequestContext ctx_, String path) {
+    AppSecRequestContext ctx = ctx_.getData(RequestContextSlot.APPSEC);
+    if (ctx == null) {
+      return NoopFlow.INSTANCE;
+    }
+    while (true) {
+      DataSubscriberInfo subInfo = ioFileSubInfo;
+      if (subInfo == null) {
+        subInfo = producerService.getDataSubscribers(KnownAddresses.IO_FS_FILE);
+        ioFileSubInfo = subInfo;
+      }
+      if (subInfo == null || subInfo.isEmpty()) {
+        return NoopFlow.INSTANCE;
+      }
+      DataBundle bundle =
+          new MapDataBundle.Builder(CAPACITY_0_2).add(KnownAddresses.IO_FS_FILE, path).build();
+      try {
+        GatewayContext gwCtx = new GatewayContext(false, RuleType.LFI);
```

Comment:
```suggestion
        GatewayContext gwCtx = new GatewayContext(true, RuleType.LFI);
```
All RASP operations should be transient